### PR TITLE
Update update_documentation.yml

### PR DESCRIPTION
**Pull request by Prasanna-Ponnusamy**
Please recheck if I added the right path

**_Describe your changes here:_**

**_All these boxes must be checked by the reviewers before merging the pull request:_**

As a reviewer please read through all the code lines and make sure that the code is fully understood, bug free, well-documented and well-structured.

- [ ] The author added a BSD statement to `doc/` (or already has one)
- [ ] The code compiles without warning in debugging and release mode, with and without MPI (this should be executed automatically in a github action)

  If the Pull request introduces code that is not covered by the github action (for example coupling with a new library):
  - [ ] Should this use case be added to the github action?
  - [ ] If not, does the specific use case compile and all tests pass (check manually)

- [ ] All tests pass (in various configurations, this should be executed automatically in a github action)
- [ ] New source/header files are properly added to the Makefiles
- [ ] New Datatypes are added to t8indent_custom_datatypes.txt
- [ ] The reviewer executed the new code features at least once and checked the results manually
- [ ] The code is covered in an existing or new test case
- [ ] New tests use the Google Test framework
- [ ] The code follows the [t8code coding guidelines](https://github.com/holke/t8code/wiki/Coding-Guideline)
- [ ] The code is well documented
- [ ] All function declarations, structs/classes and their members have a proper doxygen documentation
- [ ] All new algorithms and data structures are sufficiently optimal in terms of memory and runtime (If this should be merged, but there is still potential for optimization, create a new issue)
- [ ] If a new directory with source-files is added, it must be covered by the `script/find_all_source_files.scp` to check the indentation of these files.
- [ ] If this PR introduces a new feature, it must be covered in an example/tutorial and a Wiki article.

### CODE REVIEW

File: .github/workflows/update_documentation.yml [hunk 1]
```diff
@@ -87,7 +87,7 @@ jobs:
         cd t8code-website/content
         rm -rf doc/dev/*
         mkdir -p doc/dev
-        cp -r ../../build_debug/doxygen/html/* doc/dev/
+        cp -r ../../doc/Doxyfile/* doc/dev/
```
Comment (sandro-elsweijer) [inline .github/workflows/update_documentation.yml:90]:
```suggestion
        cp -r ../../build_debug/doc/html/* doc/dev/
```
You can find out the new path by checking out and updating your main branch and then building t8code from scratch, starting with I think the `git submodule update` command. Git will then update the SC and p4est library and adapt to the new documentation path. After configuring, you can use `make doxygen` to generate the documentation. In the old version, the html documentation was written to `<build_folder>/doxygen/html/`. With the new patch, this changes to `<build_folder>/doc/html/`.

**Review by sandro-elsweijer — CHANGES_REQUESTED** [review]
Please do the change also here:
https://github.com/DLR-AMR/t8code/blob/main/.github/workflows/add_release_documentation.yml#L125
We have one Workflow to upload the latest documentation of the main branch and one workflow to add a documentation if a new major or minor release takes place. Both are affected.

**Comment by Prasanna-Ponnusamy** [comment]
Thank you Sandro!

**Review by sandro-elsweijer — APPROVED** [review]
lgtm

**Pull request merged**